viscstudies/src/org/labkey/viscstudies/ProjectStudiesTable.java

```diff
@@ -45,9 +45,9 @@ public ProjectStudiesTable(ViscStudySchema schema, TableInfo studyTable)
         // Pretend that the project is our base container
         super(studyTable, schema.getContainer().getProject() == null ? schema : new ViscStudySchema(schema.getUser(), schema.getContainer().getProject()));
         // Set the ContainerFilter to show everything from the project
-        ContainerFilter containerFilter = new ContainerFilter.CurrentAndSubfolders(schema.getUser());
+        ContainerFilter containerFilter = ContainerFilter.Type.CurrentAndSubfolders.create(getUserSchema());
         ((ContainerFilterable)studyTable).setContainerFilter(containerFilter);
-        applyContainerFilter(containerFilter);
+        applyContainerFilter(ContainerFilter.EVERYTHING);
```

Comment:
This seems different from the original code. Is it right?

Reply:
The old code was using the same container filter _twice_ once on the inner table and once on the outer table.  So, using EVERYTHING here means we don't repeat.

Reply:
OK, probably worth a comment in the code too